src/__tests__/utils/setupTests.ts

```diff
@@ -65,7 +65,7 @@ afterEach(() => {
   jest.clearAllMocks();
   jest.resetAllMocks();
   jest.restoreAllMocks();
-  jest.runOnlyPendingTimers();
+  jest.clearAllTimers();
```

Comment:
Instead of running the pending timers (moreover outside of act !!), we're just clearing them.